### PR TITLE
Authors/About: support for multiple sections

### PR DESCRIPTION
**Pull request by ghost**
[Old screenshot](https://user-images.githubusercontent.com/26753932/27772506-e11aa0ba-5f8d-11e7-93e4-1d72fbda18b3.png)

### CODE REVIEW

**Comment by toger5** [comment]
Nice. I really don't like the horizontal lines though... Do they have any use? 
I think using spacing to make it obvious which section belongs to each title is better?

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -6042,52 +6042,73 @@ EditorNode::EditorNode() {
...
+
+		List<String> dev_sections;
+		dev_sections.push_back(TTR("Project Founders"));
+		dev_sections.push_back(TTR("Lead Developer"));
+		dev_sections.push_back(TTR("Project Manager"));
+		dev_sections.push_back(TTR("Developers"));
+
+		const char **dev_src[] = { dev_founders, dev_lead, dev_manager, dev_names };
+
+		for (int i = 0; i < 4; i++) {
```
Comment (toger5) [inline editor/editor_node.cpp]: you might want to change that to: dev_sections.size(). So if someone wants to just have more sections he does not need to increase that number too to make it work.

**Comment by Noshyaar** [comment]
I added HSeparator to reflect how section headers looked in the md file. I also don't think it works well with the editor UI. However, I find spacing is kinda tricky to add, so I just move the separator below each section and set its alpha to zero.

image

File: editor/editor_node.cpp [hunk 2]
```diff
@@ -6042,52 +6042,73 @@ EditorNode::EditorNode() {
...
+		about->set_title(TTR("Thanks from the Godot community!"));
+		about->get_ok()->set_text(TTR("Thanks!"));
+		about->set_hide_on_ok(true);
+		about->set_resizable(true);
+		gui_base->add_child(about);
+
+		VBoxContainer *vbc = memnew(VBoxContainer);
+		HBoxContainer *hbc = memnew(HBoxContainer);
+		hbc->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+		hbc->set_alignment(BoxContainer::ALIGN_CENTER);
```
Comment (toger5) [inline editor/editor_node.cpp]: what about adding? looks to tight maybe?
```hbc->add_constant_override("separation", /*someBiggerNumberThanDefault*/ 100 * EDSCALE);```

**Pull request merged**